packages/0xcert-scaffold/src/core/messages.ts

```diff
@@ -0,0 +1,91 @@
+  [ProviderIssue.ERC721_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_METADATA_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_METADATA_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_METADATA_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_METADATA_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_METADATA_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_METADATA_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_ENUMERABLE_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_ENUMERABLE_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_ENUMERABLE_INVALID_INDEX]: 'There is no asset at provided index.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_INVALID_INDEX]: 'There is no asset at provided index.',
+  [ProviderIssue.XCERT_CAPABILITY_NOT_SUPPORTED]: 'Asset ledger does not have capability for this action.',
+  [ProviderIssue.XCERT_TRANSFERS_DISABLED]: 'Transfers on this asset ledger are currently disabled.',
+  [ProviderIssue.XCERT_NOT_VALID_XCERT]: 'Asset with this id does not exist.',
+  [ProviderIssue.XCERT_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
```

Comment:
[ProviderIssue.XCERT_NOT_OWNER_NOR_OPERATOR]: 'Sender is neither asset owner nor operator.',